frontend/src/views/web/change-password.html

```diff
@@ -4,27 +4,36 @@
             <div class=" ev-card-panel ev-z-depth-5">
                 <div class="ev-md-container ev-panel-title"><strong>Change Password</strong></div>
                 <div class="ev-md-container ev-card-body change-passowrd-card">
-                    <form ng-submit="changepwd.changePassword()">
+                    <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" focus-if>
+                            <input id="old_password" name="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" required focus-if>
```

Comment:
`ng-minlength` is required here too. and don't forget to add `ng-message` to show error if length is less than 8.